src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  width: $card-width;
```

Comment:
Ensure that the SCSS variable `$card-width` is defined. This variable is crucial for the styling of the card.